public/app/features/plugins/admin/types.ts

```diff
@@ -84,6 +84,10 @@ export interface CatalogPluginDetails {
   raiseAnIssueUrl?: string;
   signatureType?: PluginSignatureType;
   signature?: PluginSignatureStatus;
+  screenshots?: Array<{
```

Comment:
declare a PluginScreenshot type and reuse it in both attributes here.
```suggestion
  screenshots?: Array<PluginScreenshot>
```